nemo_curator/modules/fuzzy_dedup/jaccardsimilarity.py

```diff
@@ -166,34 +151,29 @@ def _compute_jaccard_pair(self, docs_df, anchor_df):
             result_ls.append(result_df)
         if len(result_ls) == 0:
             return self._create_empty_jaccard_result()
-        df_pair = cudf.concat(result_ls)
-        return df_pair
+        return cudf.concat(result_ls)
 
-    def _create_empty_jaccard_result(self):
+    def _create_empty_jaccard_result(self) -> cudf.DataFrame:
         df = cudf.DataFrame()
         df[self.left_id] = "x"
         df[self.right_id] = "y"
         df["jaccard"] = np.empty(shape=0, dtype=np.float32)
         return df
 
-    def _compute_jaccard_partition(self, df):
+    def _compute_jaccard_partition(self, df: cudf.DataFrame) -> cudf.DataFrame:
         text_x = f"{self.text_field}_x"
         text_y = f"{self.text_field}_y"
         df["jaccard"] = df[text_x].str.jaccard_index(df[text_y], width=self.ngram_width)
-        df.drop(columns=[text_x, text_y], inplace=True)
+        df.drop(columns=[text_x, text_y], inplace=True)  # noqa: PD002
         return df
 
     @staticmethod
-    def _get_max_num_rows_to_process_once(df, text_field):
+    def _get_max_num_rows_to_process_once(df: cudf.DataFrame, text_field: str) -> int:
         nbytes = df[text_field].str.byte_count().sum()
         # Number of exmploded bytes
         exploded_bytes = nbytes * 5 * 2
         max_chars_allowed = 2_147_483_647
         byte_ratio = int(exploded_bytes) // max_chars_allowed
-        if byte_ratio > 1:
-            nrows_at_once = len(df) // byte_ratio
-        else:
-            nrows_at_once = len(df)
+        nrows_at_once = len(df) // byte_ratio if byte_ratio > 1 else len(df)
```

Comment:
should this be covered in paranthesis? maybe ruff removes it even if you add them
```suggestion
        nrows_at_once = len(df) // (byte_ratio if byte_ratio > 1 else len(df))
```

Reply:
I think the current way is already correct. With your way, it would divide by `byte_ratio` if it is greater than 1, else divide by `len(df)`, which is incorrect.

Reply:
Oh I see, it was the confusion I had about the behavior when we have variable = `a OPERATOR b if CONDITION else c`